devtools/maven/src/main/java/io/quarkus/maven/BuildFileMojoBase.java

```diff
@@ -147,7 +174,9 @@ private String resolveValue(String expr, BuildFile buildFile) throws IOException
         if (expr.startsWith("${") && expr.endsWith("}")) {
             final String v = buildFile.getProperty(expr.substring(2, expr.length() - 1));
             if (v == null) {
-                getLog().debug("Failed to resolve version of " + v);
+                if (getLog().isDebugEnabled()) {
+                    getLog().debug("Failed to resolve version of " + v);
```

Comment:
```suggestion
                    getLog().debug("Failed to resolve version of " + expr);
```

Because v == null :wink:

Reply:
Fixed, thanks!